include/deal.II/meshworker/scratch_data.h

```diff
@@ -0,0 +1,1301 @@
+     */
+    ScratchData(
+      const Mapping<dim, spacedim> &      mapping,
+      const FiniteElement<dim, spacedim> &fe,
+      const Quadrature<dim> &             quadrature,
+      const UpdateFlags &                 update_flags,
+      const UpdateFlags &                 neighbor_update_flags,
+      const Quadrature<dim - 1> &face_quadrature   = Quadrature<dim - 1>(),
+      const UpdateFlags &        face_update_flags = update_default,
+      const UpdateFlags &        neighbor_face_update_flags = update_default);
+
+    /**
+     * Same as the other constructor, using the default MappingQ1.
+     *
+     * @param mapping The mapping to use in the internal FEValues objects
+     * @param fe The finite element
+     * @param quadrature The cell quadrature
+     * @param update_flags UpdateFlags for the current cell FEValues
+     * @param neighbor_update_flags UpdateFlags for the neighbor cell FEValues
+     * @param face_quadrature Face quadrature, used for FEFaceFvalues and
+     * FESubfaceValues for both the current cell and the neighbor cell
+     * @param face_update_flags UpdateFlags used for FEFaceFvalues and
+     * FESubfaceValues for the current cell
+     * @param neighbor_face_update_flags UpdateFlags used for FEFaceFvalues and
+     * FESubfaceValues for the neighbor cell
+     */
+    ScratchData(
+      const FiniteElement<dim, spacedim> &fe,
+      const Quadrature<dim> &             quadrature,
+      const UpdateFlags &                 update_flags,
+      const Quadrature<dim - 1> &face_quadrature   = Quadrature<dim - 1>(),
+      const UpdateFlags &        face_update_flags = update_default);
+
+    /**
+     * Same as the other constructor, using the default MappingQ1.
+     *
+     * @param mapping The mapping to use in the internal FEValues objects
+     * @param fe The finite element
+     * @param quadrature The cell quadrature
+     * @param update_flags UpdateFlags for the current cell FEValues
+     * @param neighbor_update_flags UpdateFlags for the neighbor cell FEValues
+     * @param face_quadrature Face quadrature, used for FEFaceFvalues and
+     * FESubfaceValues for both the current cell and the neighbor cell
+     * @param face_update_flags UpdateFlags used for FEFaceFvalues and
+     * FESubfaceValues for the current cell
+     * @param neighbor_face_update_flags UpdateFlags used for FEFaceFvalues and
+     * FESubfaceValues for the neighbor cell
+     */
+    ScratchData(
+      const FiniteElement<dim, spacedim> &fe,
+      const Quadrature<dim> &             quadrature,
+      const UpdateFlags &                 update_flags,
+      const UpdateFlags &                 neighbor_update_flags,
+      const Quadrature<dim - 1> &face_quadrature   = Quadrature<dim - 1>(),
+      const UpdateFlags &        face_update_flags = update_default,
+      const UpdateFlags &        neighbor_face_update_flags = update_default);
+
+    /**
+     * Deep copy constructor. FEValues objects are not copied.
```

Comment:
```suggestion
     * Deep copy constructor.
```
The sentence I delete is a detail that is of no importance to the user who will not be able to see this difference.